packages/cli/commands/config/rename.js

```diff
@@ -0,0 +1,63 @@
+const { logger } = require('@hubspot/cli-lib/logger');
+const { renameAccount } = require('@hubspot/cli-lib/lib/config');
+const {
+  loadConfig,
+  validateConfig,
+  checkAndWarnGitInclusion,
+} = require('@hubspot/cli-lib');
+
+const {
+  addConfigOptions,
+  addAccountOptions,
+  getAccountId,
+  setLogLevel,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { logDebugInfo } = require('../../lib/debugInfo');
+const { validateAccount } = require('../../lib/validation');
+
+const loadAndValidateOptions = async options => {
+  setLogLevel(options);
+  logDebugInfo(options);
+  const { config: configPath } = options;
+  loadConfig(configPath, options);
+  checkAndWarnGitInclusion();
+
+  if (!(validateConfig() && (await validateAccount(options)))) {
+    process.exit(1);
+  }
+};
+
+exports.command = 'rename <accountName> <newName>';
+exports.describe = 'Rename account in config';
+
+exports.handler = async options => {
+  loadAndValidateOptions(options);
+
+  const { portalName, newName } = options;
```

Comment:
Looks like this should be `accountName` now. That will likely fix the issue I mentioned [here](https://github.com/HubSpot/hubspot-cli/pull/464#issuecomment-820577671).

Reply:
Fixed, good catch!